libs/service-portal/settings/personal-information/src/screens/UserProfile/UserProfile.tsx

```diff
@@ -48,7 +48,7 @@ const UserProfile: ServicePortalModuleComponent = ({ userInfo }) => {
                 labelColumnSpan={['8/12', '3/12']}
                 editColumnSpan={['1/1', '2/12']}
                 valueColumnSpan={['1/1', '7/12']}
-                content={settings?.mobile ?? ''}
+                content={settings?.mobile?.split('-')[1] ?? ''}
```

Comment:
Move into a function, this would fail if the split returns the empty array or array of length 1.
Could fix with `?.[1]` but this is getting a bit hairy for inline wrangling